charts/elastic-cloud/templates/elasticsearch_backup.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: batch/v1beta1
 kind: CronJob
 metadata:
   name: elasticsearch-snapshotter
```

Comment:
see https://github.com/rhoai/helm-charts/pull/2#discussion_r541097608 for distinct CRD metadata names.